src/js/media-controller.js

```diff
@@ -477,6 +484,28 @@ class MediaController extends MediaContainer {
         );
       },
     };
+
+    // copied from media-chrome-button
+    const keyUpHandler = (e) => {
+      const { key } = e;
+      if (!ButtonPressedKeys.includes(key)) {
+        this.removeEventListener('keyup', keyUpHandler);
+        return;
+      }
+
+      if (!this.hasAttribute('nohotkeys')) {
```

Comment:
suggestion(non-blocking): Instead of continuing to listen to irrelevant keyboard presses when `"nohotkeys"` is set, we should probably:

1. add `"nohotkeys"` to `observedAttributes`.
2. add/remove event handlers for `"keyup"`/`"keydown"` based on `attributeChangedCallback`
    1. This entails moving the event callbacks to methods instead of inline.

Reply:
updated